src/api/pack/xdsl/move/pack-xdsl-move.v6.service.js

```diff
@@ -12,6 +12,11 @@ angular.module('ovh-api-services').service('OvhApiPackXdslMoveV6', ($resource, P
       url: '/pack/xdsl/:packName/addressMove/moveOffer',
       isArray: false,
     },
+    subserviceToDelete: {
+      method: 'POST',
+      isArray: false,
+      url: '/pack/xdsl/:packName/addressMove/serviceToDelete',
```

Comment:
Endpoint is not yet available. Do you have any ETA? 

Side question: why not considering using a method `DELETE` for such an endpoint?

Reply:
The endpoint is available, it's a typo.
It's not a `DELETE` method, this method retrieve the list of services to delete updated with the list of options selected by user.